src/Storefront/Resources/views/storefront/component/product/card/box-standard.html.twig

```diff
@@ -63,7 +63,7 @@
                     {% block component_product_box_info %}
                         <div class="product-info">
                             {% block component_product_box_rating %}
-                                {% if shopware.config.detail.showReviews %}
+                                {% if shopware.config.core.listing.showReview %}
```

Comment:
it seems there were plans for a product detail setting module? maybe you could talk about that with the design

Reply:
It has to remain on the `listing` domain, so all they could do is change the key from `showReview` to `showReviews`.

Reply:
@mitelg Maybe add both conditions instead? Like `{% if shopware.config.detail.showReviews && shopware.config.core.listing.showReview%}` ?